packages/common/src/activity-options.ts

```diff
@@ -124,6 +124,14 @@ export interface ActivityOptions {
    */
   versioningIntent?: VersioningIntent;
 
+  /**
+   * A single-line fixed summary for this workflow execution that may appear in the UI/CLI.
+   * This can be in single-line Temporal markdown format.
+   *
+   * @experimental User metadata is a new API and suspectible to change.
+   */
+  staticSummary?: string;
```

Comment:
I think that should be only `summary`, no `static`. What's the terminology in other SDKs?

Reply:
a bit of a mix, but generally `static` prefix for workflow start methods, otherwise no `static`, i'ved opted for the same